### PR TITLE
Lesson 1 assignment completed

### PR DESCRIPTION
**Pull request by sallysy8**

### CODE REVIEW

**Comment by sallysy8** [comment]
Still working on the unittest of main.py.

File: students/Sally_Shen/lesson01/assignment/inventory_management/electric_appliances_class.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+"""Electric appliances class"""
+from .inventory_class import Inventory
```
Comment (nairraghav) [inline students/Sally_Shen/lesson01/assignment/inventory_management/electric_appliances_class.py]: When running, you will likely be running from the assignment directory, you will want to update this to:
from inventory_management.inventory_class import Inventory

File: students/Sally_Shen/lesson01/assignment/inventory_management/furniture_class.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+"""Furniture class"""
+from .inventory_class import Inventory
```
Comment (nairraghav) [inline students/Sally_Shen/lesson01/assignment/inventory_management/furniture_class.py]: Same import issue

**Comment by nairraghav** [comment]
-------------------------------------------------------------------
Your code has been rated at 9.05/10 (previous run: 10.00/10, -0.95)


----------------------------------------------------------------------
Ran 4 tests in 0.001s

OK
Name                                                Stmts   Miss  Cover   Missing
---------------------------------------------------------------------------------
inventory_management/__init__.py                        0      0   100%
inventory_management/electric_appliances_class.py      15      0   100%
inventory_management/furniture_class.py                15      0   100%
inventory_management/inventory_class.py                13      0   100%
inventory_management/main.py                           49     49     0%   4-93
inventory_management/market_prices.py                   2      0   100%
---------------------------------------------------------------------------------
TOTAL                                                  94     49    48%


Hi Sally,

I'm seeing the above when running pylint and the test commands. Overall, this looks good! Please try to add more coverage for the tests. I expect you are still adding main tests as per your comments. Great job so far!

**Comment by nairraghav** [comment]
Hi Sally,

I will be merging this for now, please submit a new PR for the future tests!

**Pull request merged**